contents/docs/plugins/build.md

```diff
@@ -4,17 +4,78 @@ sidebar: Docs
 showTitle: true
 ---
 
-> **Note:** It's worth familiarizing yourself with the [architecture of PostHog plugins](/docs/plugins/architecture) before building your own. 
+## 5 Minute Overview
```

Comment:
This page is now a really long list of a tutorial, overview and a reference. It might make more sense to split it out into its components, something like
- overview of how plugins work
- tutorial on what plugins are
- reference of all functions and settings

Reply:
Makes sense, I did it a bit differently, let me know if it doesn't make sense:

- Overview: How Plugins work and how the code works
- Tutorial: How to get a plugin from zero to published in our repository (this now contains explicit instructions on the developer workflow, options available, and probably the one place that will start recommending the starter kit and how to use it. I'm still working on fleshing this section out)
-  Reference for all functions